src/lm/mikolov-rnnlm-lib.cc

```diff
@@ -147,7 +147,7 @@ CRnnLM::CRnnLM() {
   srand(rand_seed);
 
   vocab_hash_size = 100000000;
-  vocab_hash  =  (int *)calloc(vocab_hash_size, sizeof(int));
+  vocab_hash  =  reinterpret_cast<int *>(calloc(vocab_hash_size, sizeof(int)));
```

Comment:
@danpovey I am not sure about this... it seems we don't really use calloc() much and use new instead. Not sure what is the right thing to do here...